tests/foreman/longrun/test_inc_updates.py

```diff
@@ -228,7 +228,7 @@ def test_positive_noapply_api(
 
 
 @pytest.mark.tier3
-def test_positive_incremental_update_time(module_target_sat, module_sca_manifest_org):
+def test_positive_incremental_update_time(target_sat, function_sca_manifest_org):
```

Comment:
@vijaysawant maybe you should use `module_entitlement_manifest_org` here.

Reply:
@shweta83 ^^

Reply:
I would always favor to entitlement for 6.15.z and older versions to be consistent in robottelo.

Reply:
same oppose for this CP as https://github.com/SatelliteQE/robottelo/pull/14842#discussion_r1576510828